widget/playground/src/containers/StyleLayout/StyleLayout.General.tsx

```diff
@@ -47,7 +47,8 @@ export function General() {
     useConfigStore.use.config().theme?.secondaryBorderRadius;
   const fontFamily =
     useConfigStore.use.config().theme?.fontFamily || DEFAULT_FONT;
-  const language = useConfigStore.use.config().language || LANGUAGES[0].value;
+  const enLanguage = LANGUAGES.find((lng) => lng.value === 'en');
```

Comment:
please don't harcode values. you can do this instead to keep one source of truth for langs: 
in widget/playground/src/constants/languages.ts
```
const DEFAULT_LANGUAGE = 'en'
 
// upate the array as well to use this
``` 

and export and use here.